src/App.tsx

```diff
@@ -1,9 +1,19 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [queryInfo, setQueryInfo] = useState('');
+  const visableMovies = moviesFromServer.filter(movie => {
+    const { description, title } = movie;
+
+    const query = queryInfo.trim().toUpperCase();
+
+    return description.toUpperCase().includes(query)
+      || title.toUpperCase().includes(query);
```

Comment:
move this logic into a function 

```suggestion
    return helperFunction(title) || helperFunction(description);
```